scripts/run_test_report.py

```diff
@@ -373,8 +378,8 @@ def generate_test_cases_from_binaries(binary_dir: str,
     parser.add_argument('--exclude_regex', type = str, help = 'Regular Expression to exclude tests that match either in the name or filter: GivenBarrier*', default = None)
     parser.add_argument('--test_run_timeout', type = int, help = 'Adjust the timeout for test binary execution, this is for each call to the binary with the filter', default = 1200)
     parser.add_argument('--log_prefix', type = str, help = 'Change the prefix name for the results such that the output is <prefix>_results.csv & <prefix>_failure_log.txt', default = "level_zero_tests")
-    parser.add_argument('--export_test_plan', type = str, help = 'Name of the Generated Test Plan to export as <arg>.csv without execution. The name provided is combined with .csv appended.', default = None)
```

Comment:
any particular reason we change how we're using the csv names?

Reply:
Because it's how I assumed it would work the first time I tried using the program, Only typing part of the file path feels weird to me. I'm being opinionated :)